_posts/2022-06-21-walkthrough.md

```diff
@@ -0,0 +1,158 @@
+
+## Installing and configuring beets
+Following are brief instructions for downloading and installing the beets CLI. For detailed instructions, head over to the official documentation here: [Getting Started](https://beets.readthedocs.io/en/stable/guides/main.html).
+
+Download beets with `pip install beets`
+
+Write your config in `~/.config/beets/config.yaml` specifying a directory for your copied music and a library for a database file to store an index of your music.
+
+My `config.yaml` looks like this:
+
+```yaml
+directory: ~/beets/music
+library: ~/beets/data/musiclibrary.db
+plugins: web
+```
+
+All I've done to support the above config is create the  `beets` directory inside my home folder, with empty child directories for `music` and `data` inside it. You can place your music library and data store wherever you like, and you don't need to create the `musiclibrary.db` file, it will be created for you.
+
+## Importing the sample library
+First, download the .zip archive of the sample music library from [this](https://bit.ly/3EZb5ue) public google drive file, extract the contents and take note of the path to the sample library for later use.
+
+Take a second to scrub through the library's organization and inconsistent file naming. On a small scale this is a prime example of how messy a music library can be in its raw form, with no standard for metadata or organization.
+
+Enter beets! We're going to clean up this mess and make our library beautiful.
+
+Run `beet import {path to sample library}` to import the messy sample library and start using the beets auto-tagger.
+
+In my case the path to the sample library is also in my home folder, so I will run the command `beet import ~/sample-library`
+
+The autotagger starts work on Album 1, and the closest match is selection `1`, go ahead and enter that in the prompt.
+
+![](beets/_posts/2022-06-21-images/album1-find-tags.png)
+
+Next, enter `a` to apply the changes found to Album 1.
+
+![](beets/_posts/2022-06-21-images/album1-correct-tags.png)
+
+Album 2 doesn't have a matching release, so we will enter one manually by entering `i` and then the following MusicBrainz id `96581fca-a7b1-4c94-bb23-7b0df9f00507`
+
+![](beets/_posts/2022-06-21-images/album2-no-release.png)
+
+Accept the changes for our manual match by selecting `a`
+
+![](beets/_posts/2022-06-21-images/album2-correct-tags.png)
+
+Album 3 doesn't have a matching release either, so we will enter one manually with `i` and then the following MusicBrainz id `95440be5-cff3-4222-8ddc-5b7ecb99a351`
+
+![](beets/_posts/2022-06-21-images/album3-no-release.png)
+
+We've located the correct album, however we only have 11/48 tracks across a compilation of 3 CDs, hence the large output warning us of missing tracks.
```

Comment:
```suggestion
We've located the correct album; however, we only have 11/48 tracks across a compilation of 3 CDs, hence the large output warning us of missing tracks.
```

Fix usage of "however" (which is [not a conjunction](https://capra.cs.cornell.edu/styleguide/#however)).